README.md

```diff
@@ -261,7 +262,7 @@ Input parameter - object of type `Service`, `Pod`, `ReplicationController`.
 The below example is for v1
 
 ```ruby
-service = Service.new
+service = Kubeclient::Resource.new
```

Comment:
The create_service call. Our long term goal is to deprecate all Kubeclient::X classes since all the implementation does with them is get their hash